File: 01-01-2026/last_letter.js
function getLastLetter(str) {
  const letters = str.match(/[a-zA-Z]/g);
  const letters_lower = letters.join("").toLowerCase();
  const char_code = letters_lower.split("").map((char) => char.charCodeAt(0));
  const max_index = char_code.indexOf(Math.max(...char_code));

  return letters[max_index];
}

console.log(get_last_letter("world")); // "w".
console.log(get_last_letter("Hello World")); // "W".
console.log(get_last_letter("The quick brown fox jumped over the lazy dog.")); // "z".
console.log(get_last_letter("HeLl0")); // "L".
console.log(get_last_letter("!#$ er@R asd fT.,> 2t0e9")); // "T".

File: 01-01-2026/last_letter.py
import re

def get_last_letter(s):
    letters = re.findall(r'[a-zA-Z]', s)
    letters_lower = "".join(letters).lower()
    char_code = [ord(char) for char in letters_lower]
    max_index = char_code.index(max(char_code))

    return letters[max_index]

print(get_last_letter("world")) # "w".
print(get_last_letter("Hello World")) # "W".
print(get_last_letter("The quick brown fox jumped over the lazy dog.")) # "z".
print(get_last_letter("HeLl0")) # "L".
print(get_last_letter("!#$ er@R asd fT.,> 2t0e9")) # "T".